scripts/OCR_Scanner/ocr.py

```diff
@@ -0,0 +1,21 @@
+from PIL import Image
+import pytesseract
+import numpy as np
+
+# Input your Local Machine Address in filename where you have stored your image.
+path = input("Enter your Image Path Address : ")
+filename = path
+img = np.array(Image.open(filename))
+text = pytesseract.image_to_string(img)
+# print(text)
+content = text
```

Comment:
Use a try except to catch exceptions in case the image path is invalid or the image itself is invalid